### PR TITLE
Updated put route to regulate changes to an community only if they are in the admin list

### PR DESCRIPTION
**Pull request by lzhahn**
This allows for admins of a community to make changes to a community and to add users as admins, U-47

### CODE REVIEW

File: tests/test_community.py [hunk 1]
```diff
@@ -142,7 +140,7 @@ def test_community_put(client):
             follow_redirects=True,
         )
         assert (
-            b'{"access_token":"AccessToken","data":{"name":"TestCommunity"},"message":"Community Updated","refresh_token":"RefreshToken","status_code":200}\n'
+            b'{"access_token":"AccessToken","data":{"Admins":["Testemail@gmail.com"],"name":"TestCommunity"},"message":"Community Updated","refresh_token":"RefreshToken","status_code":200}\n'
```
Comment (stephend017) [inline tests/test_community.py:143]: is it possible to add a second testcase that validates that when a non admin updates it fails?

**Review by stephend017 — APPROVED** [review]
LGTM. only issue is would like to see a second testcase. but the code itself looks fine.

File: tests/test_community.py (hunk 1 above)
Comment (lzhahn) [reply tests/test_community.py:143]: Yep! Added in new commit.

**Review by stephend017 — APPROVED** [review]
Looks good. Thanks for adding the new testcase

**Pull request merged**